internal/static/i18n/sv.yaml

```diff
@@ -0,0 +1,1343 @@
+            failed: lösenordsfri inloggningskontroll misslyckades
+          signcount:
+            changed: Kontrollsumman för lösenordsfri inloggningstoken har ändrats
+        initialization:
+          code:
+            added: Kod för lösenordsfri initialisering tillagd
+            sent: Kod för lösenordsfri initialisering skickad
+            requested: Kod för lösenordsfri initialisering begärd
+            check:
+              succeeded: Kontroll av kod för lösenordsfri initialisering lyckades
+              failed: Kontroll av kod för lösenordsfri initialisering misslyckades
+      signed:
+        out: Användare utloggad
+      refresh:
+        token:
+          added: Uppdateringstoken skapad
+          renewed: Uppdateringstoken förnyad
+          removed: Uppdateringstoken borttagen
+      locked: Användare låst
+      unlocked: Användare upplåst
+      deactivated: Användare avaktiverad
+      reactivated: Användare återaktiverad
+      removed: Användare borttagen
+      password:
+        changed: Lösenord ändrat
+        code:
+          added: Lösenordskod genererad
+          sent: Lösenordskod skickad
+        check:
+          succeeded: Lösenordskontroll lyckades
+          failed: Lösenordskontroll misslyckades
+      phone:
+        changed: Mobilnummer ändrat
+        verified: Mobilnummer verifierat
+        verification:
+          failed: Verifiering av Mobilnummer misslyckades
+        code:
+          added: Mobilnummerkod genererad
+          sent: Mobilnummerkod skickad
+
+      profile:
+        changed: Användarprofil ändrad
+      address:
+        changed: Användaradress ändrad
+      mfa:
+        otp:
+          added: Tvåfaktor OTP tillagd
+          verified: Tvåfaktor OTP verifierad
+          removed: Tvåfaktor OTP borttagen
+          check:
+            succeeded: Tvåfaktor OTP-kontroll lyckades
+            failed: Tvåfaktor OTP-kontroll misslyckades
+          init:
+            skipped: Tvåfaktor OTP-initialisering hoppades över
+        init:
+          skipped: Tvåfaktorinitialisering hoppades över
+      signed:
+        out: Användare utloggad
+      grant:
+        added: Auktorisering tillagd
+        changed: Auktorisering ändrad
+        removed: Auktorisering borttagen
+        deactivated: Auktorisering avaktiverad
+        reactivated: Auktorisering återaktiverad
+        reserved: Auktorisering reserverad
+        released: Auktorisering släppt
+        cascade:
+          removed: Auktorisering borttagen
+          changed: Auktorisering ändrad
+      metadata:
+        set: Användarmetadata inställd
+        removed: Användarmetadata borttagen
+        removed.all: All användarmetadata borttagen
+      domain:
+        claimed: Domän anspråkad
+        claimed.sent: Meddelande om anspråkad domän skickad
+      pat:
+        added: Personlig åtkomsttoken tillagd
+        removed: Personlig åtkomsttoken borttagen
+  org:
```

Comment:
Wrong indentation level. The following nodes belong to the `user` not `user.human`. Resulted in a yaml error.

![image](https://github.com/zitadel/zitadel/assets/5411563/75db777a-f0e3-47f5-b764-6cf41ac068ee)

```suggestion
    locked: Användare låst
    unlocked: Användare upplåst
    deactivated: Användare avaktiverad
    reactivated: Användare återaktiverad
    removed: Användare borttagen
    password:
      changed: Lösenord ändrat
      code:
        added: Lösenordskod genererad
        sent: Lösenordskod skickad
      check:
        succeeded: Lösenordskontroll lyckades
        failed: Lösenordskontroll misslyckades
    phone:
      changed: Mobilnummer ändrat
      verified: Mobilnummer verifierat
      verification:
        failed: Verifiering av Mobilnummer misslyckades
      code:
        added: Mobilnummerkod genererad
        sent: Mobilnummerkod skickad

    profile:
      changed: Användarprofil ändrad
    address:
      changed: Användaradress ändrad
    mfa:
      otp:
        added: Tvåfaktor OTP tillagd
        verified: Tvåfaktor OTP verifierad
        removed: Tvåfaktor OTP borttagen
        check:
          succeeded: Tvåfaktor OTP-kontroll lyckades
          failed: Tvåfaktor OTP-kontroll misslyckades
        init:
          skipped: Tvåfaktor OTP-initialisering hoppades över
      init:
        skipped: Tvåfaktorinitialisering hoppades över
    signed:
      out: Användare utloggad
    grant:
      added: Auktorisering tillagd
      changed: Auktorisering ändrad
      removed: Auktorisering borttagen
      deactivated: Auktorisering avaktiverad
      reactivated: Auktorisering återaktiverad
      reserved: Auktorisering reserverad
      released: Auktorisering släppt
      cascade:
        removed: Auktorisering borttagen
        changed: Auktorisering ändrad
    metadata:
      set: Användarmetadata inställd
      removed: Användarmetadata borttagen
      removed.all: All användarmetadata borttagen
    domain:
      claimed: Domän anspråkad
      claimed.sent: Meddelande om anspråkad domän skickad
    pat:
      added: Personlig åtkomsttoken tillagd
      removed: Personlig åtkomsttoken borttagen
```

Reply:
Fixed